api/v1/controllers/RestaurantsController.js

```diff
@@ -17,19 +18,36 @@ const population = {
 }
 
 exports.getRestaurants = function (req, res, next) {
-    const select = excludedFields.reduce((obj, next) => obj = {...obj, [next]: 0}, {});
-    const query = Restaurant.find({}).populate(population).select(!req.user.admin ? select : {});
+    const filter = req.query.filter;
```

Comment:
**Clarity Enhancement**
Il faudrait remplacer filter par category pour avoir des requetes tu type /restaurants?category=saucisse

Reply:
```suggestion
    const filter = req.query.category;
```
Il ne faudrait pas modifier la ligne ainsi pour être en accord avec ce que tu dis pour category ?

Reply:
Ah je viens de comprendre, je croyais que tu avais fait les modifs directement via git, my bad!